book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] In the Book class a method for adding rentals should be present, kindly make sure to create a method `add_rentals` that makes use of the `Rental` class to create a new rental